### ISSUE TITLE
Unable to start inferno locally: Latest build

### ISSUE THREAD

**Issue by ashu-bdvt** [onc-healthit/inferno#492]
Hi 
I am trying to run inferno locally against our local FHIR server. Have done that before without any issues.

However today I just cloned the latest code available and tried to run the server but getting error(pasted below) repeatedly.
```
INFO | Inferno | Checking that validator is available at http://validator_service:4567
ERROR | Inferno | Unable to reach validator at http://validator_service:4567
```
Can you tell me if something else needs to be done to make it working?

**Comment by Jammjammjamm** [comment]
Inferno uses an external service to validate FHIR resources, and with the most recent update is now checking that it can actually reach that service before starting. If you don't care about validating FHIR resources, you can change [this line of config.yml](https://github.com/onc-healthit/inferno/blob/development/config.yml#L41) to `resource_validator: internal`, but this will cause you to get strange results in any tests which perform FHIR validation.

The easiest way to get the external validator service working is using docker. If you run `docker-compose up` from the inferno directory, that should run both inferno and the validator service. If docker isn't an option, you will need to get the service running (source code: https://github.com/inferno-community/fhir-validator-wrapper), and update the [`external_resource_validator_url` in config.yml](https://github.com/onc-healthit/inferno/blob/development/config.yml#L42) to point at the url where this service is running.

**Comment by nathanloyer** [comment]
This is a duplicate of https://github.com/onc-healthit/inferno/issues/508

I commented in that issue it would be helpful if the repo made some attempt to tell us how to not run into this issue.
